connectors/cassandra-connector/src/main/scala/zio/connect/cassandra/LiveCassandraConnector.scala

```diff
@@ -0,0 +1,60 @@
+package zio.connect.cassandra
+
+import com.datastax.oss.driver.api.core.CqlSession
+import com.datastax.oss.driver.api.querybuilder.SchemaBuilder
+import zio.{Trace, ZIO, ZLayer}
+import zio.connect.cassandra.CassandraConnector.{CassandraException, CreateKeySpaceObject}
+import zio.stream.ZSink
+
+case class LiveCassandraConnector(session: CqlSession) extends CassandraConnector {
+
+  override def createKeyspace(implicit
```

Comment:
The result type looks wrong. Instead of Boolean it should be the input type. Than slot is for leftovers.